website/docs/faqs/Project/consolidate-projects.md

```diff
@@ -0,0 +1,32 @@
+---
+title: How can I consolidate projects in dbt Cloud?
+description: "Consolidating projects in dbt Cloud"
+sidebar_label: 'How to consolidate projects'
+id: consolidate-projects
+
+---
+
+Consolidating your dbt projects can be an enormous task, and there is no universal solution. But, there are some common approaches to project consolidation in dbt Cloud that you can follow, depending on the scope of the work that needs to be done.
+
+If you have multiple projects that contain production-worthy code, there are rarely straightforward solutions to merging them. Let's suppose you have `Main Project` and `Smaller Subset Project`.
+
+## Files and Folders
+
+### Git and the local directory
+
+Reference the [merge git commands](https://gist.github.com/msrose/2feacb303035d11d2d05) to help complete the migration plan. Using the commands will help retain git commit history, but you might result in duplicate folders called `models`, `tests`, etc. You will most likely still have to move files around manually.
+
+Another option would be to use an external code editor (for example, VS Code) to move files from the `Smaller Subset Project` to the `Main Project`. This is what internal dbt Labs experts recommend to stay informed about what comes over to the main project and also allows you to be more aware of the incoming files, with the ability to make any minor tweaks to folder hierarchy that you might want to do at the same time.
```

Comment:
If we recommend the second option, can we instead make it the first option? i.e. swap these lines around so we say "We recommend you use VS Code, alternatively you can also do this git switcharoo"